pages/toolkits/productivity/clickup.mdx

```diff
@@ -0,0 +1,726 @@
+# Clickup
+
+import ToolInfo from "@/components/ToolInfo";
+import Badges from "@/components/Badges";
+import TabbedCodeBlock from "@/components/TabbedCodeBlock";
+import TableOfContents from "@/components/TableOfContents";
+import ToolFooter from "@/components/ToolFooter";
+
+<ToolInfo
+  description="Enable agents to interact with Clickup"
+  author="Arcade"
+  authType="OAuth2"
+  versions={["1.0.0"]}
+/>
+
+<Badges repo="arcadeai/arcade_clickup" />
+
+The ClickUp toolkit provides tools to interact with ClickUp workspaces, projects (spaces/folders/lists), tasks, comments, and members. It enables building agents and apps that can:
+
+- Discover workspace structure and users (WhoAmI, GetWorkspaceInsights, GetSpaces, GetFoldersForSpace, GetListsForSpace, GetListsForFolder, GetMembersForWorkspace).
```

Comment:
nit: I don't think we need to list the tools here, it's just an intro supposed to explain the main capabilities. Right below it is the table of contents listing all tools already.